x/staging/posts/client/cli/query.go

```diff
@@ -219,7 +219,9 @@ func GetCmdQueryReports() *cobra.Command {
 			return clientCtx.PrintProto(res)
```

Comment:
The request of query cmd should add pagination as well. such like:
```go
pageReq, err := client.ReadPageRequest(cmd.Flags())
	if err != nil {
		return err
	}

	res, err := queryClient.Reports(
		context.Background(),
		&types.QueryReportsRequest{PostId: args[0], Pagination: pageReq},
	)
```

Reply:
whoops! Forgot it, Thanks! @dadamu